akka-stream/src/main/protobuf/StreamRefContainers.proto

```diff
@@ -12,17 +12,11 @@ option optimize_for = SPEED;
 // TODO make explicit types for SinkRef/SourceRef as they can carry initial settings (initial demand etc)
 
 message SinkRef {
-    optional ActorRef targetRef = 1;
-    // this will be basically always false, since we already materialized this "connection"
-    // so materializing another Source, where this Sink was created BY materializing a Source does not make sense
-    optional bool canMaterializeSource = 2;
+    ActorRef targetRef = 1;
```

Comment:
Protobuf files need to be regenerated, I didn't try to make that work on my machine.

Reply:
Whoop, will do

Reply:
I think the required keyword is needed tho? Yes, excellent that we can have it required :)